tests/release/pipelines/fbc_release.go

```diff
@@ -346,3 +326,19 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 	})
 	Expect(err).NotTo(HaveOccurred())
 }
+
+func fwRefresh(workspace string) *framework.Framework {
```

Comment:
nit: based on what this function does, perhaps `newFramework` would suit it better?

Reply:
It's just to reduce some repeatable code in the file.

Reply:
yes, the comment is about changing the function name

Reply:
Good name! Updated.